types/integra/aquastream/default.schema.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$id": "https://akenza.io/integra/aquastream/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "volume": {
+      "title": "Volume",
+      "unit": "L",
+      "type": "number",
+      "description": "Total volume in liter"
+    },
+    "volumeM3": {
```

Comment:
is there a description when cubic meters vs liters are used? if yes try to rename the data key according to this description